Rocket.Unturned/Commands/CommandTpwp.cs

```diff
@@ -58,19 +59,19 @@ public void Execute(IRocketPlayer caller, string[] command)
             }
 
             Vector3 position = player.Player.quests.markerPosition;
-            if (!raycastFromSkyToPosition(ref position))
+            if (!RaycastFromSkyToPosition(ref position))
             {
                 UnturnedChat.Say(player, U.Translate("command_tpwp_failed_raycast"));
                 throw new WrongUsageOfCommandException(caller, this);
             }
-
-            player.Teleport(new Vector3(position.x, position.y, position.z), player.Rotation);
-            Core.Logging.Logger.Log(U.Translate("command_tp_teleport_console", player.CharacterName, position.x + "," + position.y + "," + position.z));
-            UnturnedChat.Say(player, U.Translate("command_tp_teleport_private", position.x + "," + position.y + "," + position.z));
+            //player.Teleport(new Vector3(position.x, position.y, position.z), player.Rotation);
+            player.Player.teleportToLocationUnsafe(new Vector3(position.x, position.y, position.z), player.Rotation);
+            Core.Logging.Logger.Log(U.Translate("command_tp_teleport_console", player.CharacterName, position.x + ", " + position.y + ", " + position.z));
+            UnturnedChat.Say(player, U.Translate("command_tp_teleport_private", position.x + ", " + position.y + ", " + position.z));
         }
 
         // Copy of Unturned function because its protected
-        protected static bool raycastFromSkyToPosition(ref Vector3 position)
+        protected static bool RaycastFromSkyToPosition(ref Vector3 position)
```

Comment:
rename `RaycastFromSkyToPosition` to `raycastFromSkyToPosition` as before, just in case to keep backward compatibility